api/prove.js

```diff
@@ -132,15 +136,43 @@ switch (options.inputgen || options.test || options.prove) {
 
   // Prove mode
   case options.prove === true:
-    let result = await zkgapi.prove(
-      wasmUnit8Array,
+    const feeInWei = ethers.utils.parseEther("0.005");
+    const provider = new ethers.providers.JsonRpcProvider(
+      config.DispatcherProviderUrl
+    );
+    const signer = new ethers.Wallet(config.UserPrivateKey, provider);
+
+    let dispatcherContract = new ethers.Contract(
+      config.DispatcherContract,
+      TdABI,
+      provider
+    ).connect(signer);
+
+    const md5 = ZkWasmUtil.convertToMd5(wasmUnit8Array).toUpperCase();
+    const tx = await dispatcherContract.prove(
+      md5,
       privateInputStr,
       publicInputStr,
-      config.ZkwasmProviderUrl,
-      config.UserPrivateKey,
-      enableLog,
+      {
+        value: feeInWei,
+      }
     );
 
```

Comment:
can we add a log before and after tx.wait?
e.g. "Prove Request Transaction Sent, Waiting for Confirmation ... xxx {tx hash}"
"Transaction Confirmed. Creating Prove Task ...".